sherpa/plot/pylab_backend_area.py

```diff
@@ -36,10 +36,10 @@
 class PylabErrorArea(PylabBackend):
     '''A Matplotlib backend displaying data uncertainties as shaded regions
 
-    This class changes the bahavior of the plotting in a way that is not
+    This class changes the behavior of the plotting in a way that is not
     possible with just setting parameters alone: For 1D data with y-errors
-    the error range is not shown with error bars as in `PylabBackend`, but
-    instead with shaded regions.
+    the error range is not shown with error bars as in 
+    `sherpa.plot.backends.PylabBackend`, but instead with shaded regions.
```

Comment:
I think this should be `sherpa.plot.pylab_backend.PylabBackend` ?

Reply:
I was part of the same file initially when a local link should have worked, but then I moved it, so yes, I know need an absolute link.

Reply:
Ive un-resolved this as I was just about to complain about exactly this symbol.